homeassistant/components/sensor/yweather.py

```diff
@@ -1,6 +1,5 @@
 """
 Support for the Yahoo! Weather service.
-
```

Comment:
It's look like a mistake, didn't realize that it was there until the code was compared, could be removed if it makes a problem